### PR TITLE
feat: support current_utctimestamp

### PR DESCRIPTION
**Pull request by johannes-vogel**
works locally with SAP HANA Cloud but does not work with HXE

### CODE REVIEW

File: db-service/lib/cqn2sql.js [hunk 1]
```diff
@@ -990,7 +990,7 @@ class CQN2SQLRenderer {
     } else {
       cds.error`Invalid arguments provided for function '${func}' (${args})`
     }
-    const fn = this.class.Functions[func]?.apply(this.class.Functions, args) || `${func}(${args})`
+    const fn = this.class.Functions[func]?.apply(this.class.Functions, args) || `${func}${args ? `(${args})`: ''}`
```
[the remark this answers is not in the document]
Comment (johannes-vogel) [reply db-service/lib/cqn2sql.js:993]: revert

File: hana/test/hana-functions.test.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+const cds = require('../../test/cds')
```
Comment (BobdenOs) [inline hana/test/hana-functions.test.js:1]: Should we add this function definition into the other databases? 
Then the test can be move here:
https://github.com/cap-js/cds-dbs/blob/45bcdcf30ed103c1db8283d30ce5d84a7748a1d5/test/compliance/functions.test.js#L339-L348

Comment (johannes-vogel) [reply hana/test/hana-functions.test.js:1]: I initially started there but the other databases don't support the argument

Comment (johannes-vogel) [reply hana/test/hana-functions.test.js:1]: https://github.com/cap-js/cds-dbs/pull/909 but I don't like the different behavior on the dbs

**Review by BobdenOs — APPROVED** [review]

File: hana/test/hana-functions.test.js (hunk 2 above)
Comment (BobdenOs) [reply hana/test/hana-functions.test.js:1]: It is a good finding that the databases have different response formats. The purpose of the cal-functions is to define them in a way that they response in the correct format. Most likely adding a `cast` or `element` to the original func can apply the correct output converter for each database.

**Pull request merged**